src/main.py

```diff
@@ -0,0 +1,41 @@
+import telebot
+import xlrd
+from random import (randint, random)
+def rand():
+    rb = xlrd.open_workbook('/Users/admin/Projects/linguahelper/Saved translations.xlsx')
+    sheet = rb.sheet_by_index(0)
+    rownum = randint(0, sheet.nrows)
+    return sheet.cell_value(rownum, 0)+' ----- '+sheet.cell_value(rownum, 1) + '\n' +sheet.cell_value(rownum, 2)+ ' ----- ' + sheet.cell_value(rownum, 3)
+    """Эта функция  выбирает случайного слова с Exsel файла."""
```

Comment:
- docstring должен быть на английском
- docstring всегда пишется вначале функции

Пример:
```suggestion
def check_payload_source(payload, table_name):
    """
    Check if the payload (i.e, content of the script) of the recipe to execute doesn't contains any reference to
    preproduction tables
    Parameters
    ----------
    payload: `str`
        Payload of the recipe to execute
    table_name: `str`
       Name of table to deploy
    Returns
    -------
    `str`
        Archived version of the previous production table. Can be useful for post deployment functions to execute
        after deployment
    """

```